test/e2e/e2e_test.go

```diff
@@ -7,7 +7,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+	http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
The license header check doesn't like this change: https://github.com/submariner-io/submariner-operator/actions/runs/3281391453/jobs/5403336419#step:4:106